src/components/Element/index.js

```diff
@@ -25,12 +25,15 @@ export default {
     themeable,
   ],
   methods: {
-    renderElement(tag, options, items, parentable) {
+    renderElement(tag, options, items, parentable, notSelectable) {
```

Comment:
I'm trying to figure out this: if an element doesn't need to be clickable inside Builder, do we even need schema attributes for it? If not, we could use regular renderElement in that case (From CForm), instead of this one, and keep this method a bit simpler.